lib/ansible/modules/cloud/misc/virt.py

```diff
@@ -431,6 +441,7 @@ def define(self, xml):
 def core(module):
 
     state      = module.params.get('state', None)
+    autostart  = module.params.get('autostart', False)
```

Comment:
A default value is already defined in [`argument_spec`](https://github.com/nerzhul/ansible/blob/f38cfdee6c9329ce4ea09ea9983796fcfc3e36f0/lib/ansible/modules/cloud/misc/virt.py#L527), it is unnecessary to indicate a default value here.